tests/e2e/src/pages/base.page.ts

```diff
@@ -280,4 +280,15 @@ export class BasePage {
       await expect(result).toBe(false);
     }
   }
+  async checkElementToContainText(elementType: string, value: string, checkText: string) {
+    element = await this.returnElementByType(elementType, value);
+
+    if (elementType === "testId" && value === "warning_modal") {
+      const element = `//*[@class='modal-card']//*[contains(., '${checkText}')]`;
```

Comment:
@abilevych can you please pay attention to this xpath record. It works for us, but takes a huge field of potential records. Can we use it? `//text()[contains(., 'No tokens')]` - this xpath takes exact string but works only locally.